web/components/SessionCard/SessionCard.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  formatDateWithDay,
+  formatTimeDuration,
+  formatTimeRange,
+  getNonLocationTimezone,
+} from '../../util/date';
+import styles from './SessionCard.module.css';
+
+interface SessionCardProps {
+  title: string;
+  startTime?: Date;
+  duration?: number;
+  timezone?: string;
+}
+
+export const SessionCard = ({
+  title,
+  startTime,
+  duration,
+  timezone,
+}: SessionCardProps) => (
+  <div className={styles.card}>
+    <strong className={styles.title}>{title}</strong>
+    {startTime && (
+      <div>
+        <time dateTime={startTime.toString()}>
```

Comment:
`toString` does not give a format that is valid here.

Originally it was implementation-dependent; now it'll give something like (depending on your timezone, and whether it's server-rendered or on the client)

```<time dateTime="Tue May 24 2022 19:30:00 GMT+0000 (Coordinated Universal Time)">Tuesday – May 24</time>```

HTML requires a particular format, which in this case should be a "[valid global date and time string](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-global-date-and-time-string)".

```suggestion
        <time dateTime={startTime.toISOString()}>
```